pkg/config/config.go

```diff
@@ -168,7 +173,7 @@ in a user friendly format`, binName, binName)
 	if userConfig.Timeout <= 0 {
 		return userConfig, errors.New("you must provide a timeout greater than 0")
 	}
-	fmt.Fprintf(outputStream, "UserConfig: %v\n", userConfig)
+	log.Printf("Starting Instance-Qualifier with User Config: %s\n", userConfig.String())
```

Comment:
Does this write it to a log file instead of the cli now?

Reply:
No, still prints in the terminal (there's currently no log file for IQ itself):

```

➜  amazon-ec2-instance-qualifier git:(improve-cx) ✗ ./build/ec2-instance-qualifier --config-file=/Users/crtbry/workplace/go/src/iq_brycahta_fork_ws/amazon-ec2-instance-qualifier/2_instances.json
2020/10/09 10:46:28 Starting Instance-Qualifier with User Config: 
                InstanceTypes: m4.large,
                TestSuiteName: /Users/crtbry/workplace/go/src/iq_brycahta_fork_ws/amazon-ec2-instance-qualifier/test/app-for-e2e/tests,
                CpuThreshold: 80,
                MemThreshold: 10,
                VpcId: ,
                SubnetId: ,
                AmiId: ,
                Timeout: 600,
                Persist: false,
                Profile: ,
                Region: us-east-2,
                Bucket: ,
                CustomScriptPath: /Users/crtbry/workplace/go/src/iq_brycahta_fork_ws/amazon-ec2-instance-qualifier/test/templates/custom-script_sample.template,
                ConfigFilePath: /Users/crtbry/workplace/go/src/iq_brycahta_fork_ws/amazon-ec2-instance-qualifier/2_instances.json

```